Source/JavaScriptCore/bytecode/CallLinkInfo.h

```diff
@@ -527,13 +522,19 @@ class OptimizingCallLinkInfo final : public CallLinkInfo {
         return m_frameShuffleData.get();
     }
 
+    CodeOrigin codeOrigin() const { return m_codeOrigin; }
+
     void initializeFromDFGUnlinkedCallLinkInfo(VM&, const DFG::UnlinkedCallLinkInfo&);
 
 private:
     MacroAssembler::JumpList emitFastPath(CCallHelpers&, GPRReg calleeGPR, GPRReg callLinkInfoGPR) WARN_UNUSED_RETURN;
     MacroAssembler::JumpList emitTailCallFastPath(CCallHelpers&, GPRReg calleeGPR, GPRReg callLinkInfoGPR, ScopedLambda<void()>&& prepareForTailCall) WARN_UNUSED_RETURN;
 
-    CodeLocationNearCall<JSInternalPtrTag> m_callLocation;
+    CodeOrigin m_codeOrigin;
+    CodeLocationNearCall<JSInternalPtrTag> m_callLocation NO_UNIQUE_ADDRESS;
+    GPRReg m_calleeGPR { InvalidGPRReg };
+    GPRReg m_callLinkInfoGPR { InvalidGPRReg };
```

Comment:
These `GPRReg` are `uint8_t`.  Are there savings to be had if we move them above `m_codeOrigin`?

Reply:
Not much. We attached NO_UNIQUE_ADDRESS so that we are putting them into the padding.